index.js

```diff
@@ -25,18 +32,27 @@ var DEVICE_IDS_AVAILABLE = 'idsAvailable';
  * @param {Object}		options
  * @param {function}	options.onNotificationOpened - Fired when a remote notification is received.
  * @param {function} 	options.onError - None
- */
+ */             
 Notifications.configure = function(options: Object) {
 	if ( typeof options.onError !== 'undefined' ) {
 		this.onError = options.onError;
 	}
 
+	if ( typeof options.onNotificationReceived !== 'undefined' ) {
+		this.onNotificationReceived = options.onNotificationReceived;
+
+		if (_pendingNotifications.length > 0) {
```

Comment:
Typo: you probably meant `_pendingNotificationsReceived` here instead of `_pendingNotifications`.